symengine/ruby/valgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=memcheck --num-callers=15 --partial-loads-ok=yes --undef-value-errors=no ruby -S rspec spec
```

Comment:
Also document how it should be used and what the difference is from the other valgrind script.